test/validators/movie.test.js

```diff
@@ -44,12 +46,78 @@ describe('movie validator', () => {
         title: 'foo',
         release_year: 12345
       };
-      const result = Joi.validate(payload, MovieValidator);
+      const result = Joi.validate(payload, MovieValidator.payload);
 
       expect(result.error.details[0].path).to.eql('release_year');
       expect(result.error.details[0].type).to.eql('number.max');
     });
 
   });
 
+  describe('release_year_start', () => {
+
+    it('is after 1878', () => {
+      const query = {
+        title: 'poop',
+        release_year_start: 1776
```

Comment:
Actually in Joi, min(1878) the min value is allowed. So 'it is after 1878' is not the best wording. ... I didn't write it haha